composer.json

```diff
@@ -18,12 +18,17 @@
         "symfony/console": "^3.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "*"
+        "phpunit/phpunit": "*",
+        "squizlabs/php_codesniffer": "^2.8"
```

Comment:
@adam-paterson ah brilliant, big fan of them, use it in most projects. Really improves overall quality